README.md

```diff
@@ -218,17 +225,57 @@ const layout = new Layout({
 
 The Layout instance has the following API:
 
-### .middleware()
+### .process(HttpIncoming)
 
-A connect compatible middleware which takes care of multiple operations needed for
-a Layout to fully work.
+Metod for processing a incomming http request. This method is intended to be
```

Comment:
incoming

Reply:
Method for processing an incoming HTTP...